.gitignore

```diff
@@ -50,4 +50,7 @@ docs/Gemfile.lock
 yarn.lock
 
 # Generated binaries
-**/bin
\ No newline at end of file
+**/bin
+
+cmd/errorutil/errorutil
+/errorutil
```

Comment:
```suggestion
**errorutil_analyze_errors.json
**errorutil_analyze_summary.json
**errorutil_errors_export.json
```

Reply:
Also, you'll need to delete these files so that git stops tracking them.

Reply:
please confirm 
i dont see any of those three files anywhere in the repo

Reply:
Yes. Good catch @Shreyas220. They're not present rn. But we don't want someone accidentally adding them when they run the utility. So we need this here.

Reply:
so ill just make those 3 files in helpers